blocks/sidenav/sidenav.js

```diff
@@ -74,6 +73,62 @@ const TEMPLATE = /* html */`
       </div>
   </aside>`;
 
+/**
+ * Add version dropdown
+ * @param {Element} wrapper
+ */
+function initVersionDropdown(wrapper) {
+  const versionsDropdown = wrapper.querySelector('.version-dropdown');
+  if (!store.product) {
+    versionsDropdown.remove();
+    return;
+  }
+
+  const { lang } = document.documentElement;
+  const curVersionKey = getMetadata('version');
+  const versionsDropdownMenu = versionsDropdown.querySelector('.version-dropdown-menu ul');
+
+  versionsDropdown.addEventListener('mouseenter', async () => {
+    const json = await store.fetchJSON(`${PATH_PREFIX}/${lang}/versions`, store.product);
+    if (!json) return;
+
+    const curVersion = json.data.find((row) => row.Key === curVersionKey);
+
+    const { pathname } = window.location;
+    // rm leading slash, lang, product
+    let segments = pathname.substring(PATH_PREFIX.length).split('/').slice(3);
+    // if current href has version folder, remove it
+    if (curVersion.Folder) {
+      segments = segments.slice(1);
+    }
+    const unversionedPath = segments.join('/');
+
+    // TODO: This still doesn't quite work since the book directory names also change
```

Comment:
@maxakuru  Do we still need to support a pathname without version (ie no version = most recent version) ?

Reply:
it seems that some "most recent" versions are still in version directories in docs, so atm we need the ability to specify which versions are in folders.. we're deciding on the path format soon though, hopefully to do a consistent one of these:
1. `product/book/version`
2. `product/version/book`

where `book` will be the same name across all versions, and `version` either exists for all versions or never exists for most recent versions

Reply:
this is resolved now, format is `product/book/version`, docs are moved to fit that expectation and `/en/compute/admin-guide-pcee` is now `/en/compute/pcee/admin-guide`

for pages that shouldn't show a version picker (ie. non-versioned books like `/en/compute/is-notifications`) we'll use the `version` metadata `not-applicable`